### ISSUE TITLE
C-] opens the definition on an existing buffer and not the current buffer

### ISSUE THREAD

**Issue by dsdeiz** [shawncplus/phpcomplete.vim#69]
For some weird reason <kbd>C-]<kbd> opens a definition on an existing buffer and not the current buffer. Is this an expected behavior?

**Comment by complex857** [comment]
Do you mean (this is my best guess) that if you have the file with the definition already in a buffer then it somehow makes a difference when hitting the <kbd>c-]</kbd> combination compared to when you don't?

I'm not sure what do you mean by existing vs current buffer. Could you please ellaborate on this?

**Comment by dsdeiz** [comment]
Oops, sorry. This is how I can reproduce it:

1. Open a file which is now on window A.
2. <kbd>C-W C-]</kbd> on a function (at this time it opens the file on window B and goes to the definition - somewhat messed up my terms, should've been window instead of buffer)
3. On window B, <kbd>C-]</kbd> on another function. This time it opens the function definition on window A.

I was assuming on the 3rd step, it would simply open the file with the function definition on window B but instead it opened it on window A.

**Comment by complex857** [comment]
Oh, i see. This jump-to-definition feature works by vim's own [:tag](http://vimdoc.sourceforge.net/htmldoc/tagsrch.html#:tag) command (prefixed with the number) so i guess that this is how it's supposed to work, but would worth checking if the behaviour changes when the plugins is disabled.

**Comment by dsdeiz** [comment]
Yeah it's different without the plugin. On the 3rd step, it would open on window B when pressing <kbd>C-]</kbd> on a function call in window B. Not sure if it's relevant but `:map <C-]>` produces a call to `phpcomplete#JumpToDefinition('normal')`.

**Comment by complex857** [comment]
Thanks for looking into this for me!
At this moment i have no clue of the reasons behind this, but I'll try to get the bottom of this.
